playbooks/openstack-osh_hostconfig.yml

```diff
@@ -19,19 +20,12 @@
         - SUSE-CA
         - OpenStack-Cloud8-product
         - SLE12Containers
-      tags:
-        - skip_ansible_lint
 
-    # TODO(evrardjp): Revert back to zypper module when dist-upgrade is fixed there
-    - name: Refresh zypper. After this step, we can rely on ansible modules.
-      raw: |
-        #TODO(evrardjp): Check if utilities/devel tools/virt/cloud8updates are really needed
-        #zypper ar -f -G https://provo-mirror.opensuse.org/repositories/utilities/SLE_12/ SLE12-utilities;
-        #zypper ar -f -G https://provo-mirror.opensuse.org/repositories/Virtualization:/containers/SLE_12_SP3/ Virtualization_containers;
-        #zypper ar -f -G  http://kirk.provo.novell.com/repositories/SUSE/Products/OpenStack-Cloud/8/x86_64/product/ openstackcloud-8-update
-        zypper refresh && zypper dist-upgrade -y
-      tags:
-        - skip_ansible_lint
+    - name: Zypper dist-upgrade
+      zypper:
+        name: '*'
+        state: dist-upgrade
+        update_cache: yes
```

Comment:
could you add `extra_args_precommand: "--gpg-auto-import-keys"` ? I am testing this right now, as the key is not trusted after import on the previous step.

Reply:
Worked with extra_args_precommand.

Reply:
done!